woodwork/statistics_utils.py

```diff
@@ -147,14 +148,13 @@ def _get_describe_dict(
         if extra_stats:
             if column.is_numeric:
                 values["histogram"] = _get_histogram_values(series, bins=bins)
-                if isinstance(column.logical_type, (Integer, IntegerNullable)):
-                    _range = range(int(values["min"]), int(values["max"]) + 1)
-                    # Calculate top numeric values if range of values present
-                    # is less than or equal number of histogram bins
-                    if len(_range) <= bins:
-                        values["top_values"] = _get_numeric_value_counts_in_range(
-                            series, _range
-                        )
+                _range = range(int(values["min"]), int(values["max"]) + 1)
+                # Calculate top numeric values if range of values present
+                # is less than or equal number of histogram bins
```

Comment:
This can probably be updated to explain what the `series % 1 == 0` check is doing.

Reply:
Updated!